optuna/integration/dask.py

```diff
@@ -99,9 +98,13 @@ def create_new_study(
         self,
         comm: "distributed.comm.tcp.TCP",
         storage_name: str,
+        directions: List[str],
```

Comment:
I think so, but the original version of dask's `_OptunaSchedulerExtension.set_study_directions` takes `List[str]`, and if I change it will be inconsistent with `get_study_directions` which returns `List[str]`.

Would it be better to change these types?

Reply:
I would say it's better to use `List[str]` here since each storage implementation would be a bit simpler by restricting the type.